server/data/data-jmap-postgres/src/main/java/org/apache/james/jmap/postgres/change/PostgresMailboxChangeDAO.java

```diff
@@ -62,6 +63,14 @@ public Mono<Void> insert(MailboxChange change) {
             .set(CREATED, toUUIDArray(change.getCreated()))
             .set(UPDATED, toUUIDArray(change.getUpdated()))
             .set(DESTROYED, toUUIDArray(change.getDestroyed()))
+            .set(DATE, change.getDate().toOffsetDateTime())
+            .onConflictOnConstraint(PRIMARY_KEY_CONSTRAINT_NAME)
```

Comment:
The state value is 
`UuidCreator.getTimeOrderedEpoch()`
I don't think it needs to "do Update on Conflict...", It looks like pr tried to fix it for stable testing, not for real-world

Reply:
changed to do nothing